acct-group/bitcoin/bitcoin-0.ebuild

```diff
@@ -0,0 +1,9 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-group
+
+DESCRIPTION="System-wide Bitcoin services user"
```

Comment:
s/user/group/ at least, but it seems most packages have some description of the form "A group for the Spectrum messaging transport"

Reply:
"A group for users with access to system-wide Bitcoin services"